File: help/eng_gb/exercicios_da_ficha_7/Ficha07_009/Ficha07_009.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha07_009 {

    //Main Function
    static void Main() {
        Console.Write("x : " );
        int x = int.Parse(Console.ReadLine());
        Console.Write("f(" + x + ")= " );
        if(x % 2 == 0 ){
            x++;
        }
        Console.Write(x );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 22:37

File: help/eng_gb/exercicios_da_ficha_7/Ficha07_009/Ficha07_009.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha07_009 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("x : " );
        int x = Integer.valueOf(keyb.nextLine());
        System.out.print("f(" + x + ")= " );
        if(x % 2 == 0 ){
            x++;
        }
        System.out.print(x );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 22:37
